cl/alerts/migrations/0010_add_schedule_alert_hit_content_type.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 5.0.6 on 2024-07-18 01:45
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("alerts", "0009_alter_alert_search_types_noop"),
+        ("contenttypes", "0002_remove_content_type_name"),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="scheduledalerthit",
+            name="content_type",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="contenttypes.contenttype",
+            ),
+        ),
+        migrations.AddField(
+            model_name="scheduledalerthit",
+            name="object_id",
+            field=models.PositiveIntegerField(null=True),
+        ),
+        migrations.AddIndex(
+            model_name="scheduledalerthit",
+            index=models.Index(
+                fields=["content_type", "object_id"],
+                name="alerts_sche_content_c5e627_idx",
+            ),
+        ),
```

Comment:
Since both fields are new, the index creation process should be relatively quick. however, to make sure we don't slow down anything else while it's working, let's use `AddIndexConcurrently`.

Reply:
Done! I split the migrations into two files: one for adding the new fields and another for the concurrent index, since it needs to be outside of a transaction (atomic = False)